README.md

```diff
@@ -9,6 +9,15 @@ C API to Qt for Qlift
 - `ENABLE_SHARED`: Enable or disable building qlift-c-api as a shared library. Default is ON.
 - `ENABLE_STATIC`: Enable or disable building qlift-c-api as a static library. Default is OFF.
 
+To install as a shared library in ```/usr/local```:
+```sh
+git clone https://github.com/Longhanks/qlift-c-api
+cd qlift-c-api
+cmake CMAKE_INSTALL_PREFIX=/usr/local .
```

Comment:
You'll have to pass options to cmake via the -D flag (e.g. `cmake -DCMAKE_INSTALL_PREFIX=/usr/local`). The fact that this still installs to `/usr/local` is because that's the default. You can try this by changing the prefix, it will still install to /usr/local.

Also, I'd recommend to create a seperate build directory, cd into that, and build out of the source tree so that the sources are kept clean (e.g. git clone, cd qlift-c-api, mkdir build, cd build, cmake -DFLAG1 -DFLAG2 ..).